pyanaconda/geoloc.py

```diff
@@ -489,52 +501,56 @@ def refresh(self, force=False):
         # check if refresh is needed
         if force is True or self._result is None:
             log.info("Starting geolocation lookup")
-            log.info("Geolocation provider: %s", self.get_name())
-            global refresh_in_progress
-            with refresh_condition:
-                refresh_in_progress = True
+            log.info("Geolocation provider: %s", self.name)
+            with self._refresh_condition:
+                self._refresh_in_progress = True
 
             start_time = time.time()
             self._refresh()
             log.info("Geolocation lookup finished in %1.1f seconds",
                      time.time() - start_time)
 
-            with refresh_condition:
-                refresh_in_progress = False
-                refresh_condition.notify_all()
+            with self._refresh_condition:
+                self._refresh_in_progress = False
+                self._refresh_condition.notify_all()
             # check if there were any results
-            result = self.get_result()
-            if result:
+            if self.result:
                 log.info("got results from geolocation")
-                slog.info("geolocation result:\n%s", result)
+                slog.info("geolocation result:\n%s", self.result)
```

Comment:
Please rename `slog`. Not all abbreviations are great and I personally wouldn't guess that `slog` means `sensitive_log`. I would rather see `sensitive_log` and type little bit more than trying to find what `slog` is.